scripts/update/lidarr.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+if [[ -f /install/.lidarr.lock ]]; then
+
+    #Move old homedirectory installations to opt and switch to netcore
+    user=$(cut -d: -f1 < /root/.master.info)
+    if [[ -d /home/$user/Lidarr ]]; then
+        echo_info "Moving Lidarr instllation to opt and switching it to net-core"
+        wasActive=$(systemctl is-active lidarr)
+        systemctl stop lidarr
+
+        mv /home/"$user"/Lidarr /opt/Lidarr
```

Comment:
replace with rm -rf /home/"$user"/Lidarr

we need to remove /opt/Lidarr when upgrading to netcore anyway